site/docs/v1/tech/plugins/custom-password-hashing.adoc

```diff
@@ -111,6 +111,11 @@ If you are looking to write your own custom password hashing algorithm, this pro
 There is also a https://github.com/FusionAuth/fusionauth-contrib/tree/master/Password%20Hashing%20Plugins[selection of contributed plugins], provided by the community and made available without warranty.
 That may also be useful to you, as someone may already have written the hasher you need.
 
+[NOTE]
+====
+All sample hashers have been moved over with the community contributed plugins.
```

Comment:
```suggestion
All community contributed hashing plugins have been moved over to the https://github.com/FusionAuth/fusionauth-contrib/[fusionauth-contrib project].
```

Reply:
Do you want to mention the example code? 

https://github.com/FusionAuth/fusionauth-example-password-encryptor

or is that overkill.

Reply:
I think that is referenced up on line 108. The confusion was about the Myexample not actually being in the password-encryptor repo because it has been moved.